Multisig.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -199,6 +199,15 @@
           "version": "1.30.2"
         }
       },
+      {
+        "package": "Starscream",
+        "repositoryURL": "https://github.com/daltoniam/Starscream.git",
```

Comment:
do we really need it? Every 3rd party library brings additional risk. This one had it's latest release in 2019. We might get stuck at some point due to the lack of maintenance for it.

Reply:
I will do in next pr